lib/util/git/auth.spec.ts

```diff
@@ -28,6 +28,45 @@ describe('util/git/auth', () => {
       });
     });
 
+    it('returns url with username and password', () => {
+      expect(
+        getGitAuthenticatedEnvironmentVariables('https://github.com/', {
```

Comment:
don't do that for GitHub.com. password auth is not working there. it was disabled long time ago.

Reply:
username:password auth does work for github.com/GitHub Enterprise Server as long as the password is an access token. So you can technically supply Renovate with a `usernane: some-user`, `password: ghp_token` and it will work